frontend/package.json

```diff
@@ -11,14 +11,15 @@
     "lodash": "^4.17.15",
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
+    "react-calendar": "^3.0.1",
     "react-credit-cards": "^0.8.0",
-    "react-calendar": "^3.0.0",
     "react-dom": "^16.12.0",
     "react-router-dom": "^5.1.2",
     "react-router-prop-types": "^1.0.4",
     "react-scripts": "^3.4.0",
     "semantic-ui-css": "^2.4.1",
-    "semantic-ui-react": "^0.88.2"
+    "semantic-ui-react": "^0.88.2",
+    "yarn": "^1.22.4"
```

Comment:
Can remove this by running npm uninstall yarn. We gonna stick to using npm only to reduce confusion.